certified-connectors/Oodrive Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2891 @@
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/contracts/{id}/transaction/recipients": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "List of signature informations",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/RecipientForContractWithRefusal"
+              }
+            }
+          },
+          "404": {
+            "description": "Contract not found",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "string",
+                  "description": "Error message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get signatures of a contract",
+        "description": "List all the signatures informations from a specific contract",
+        "operationId": "ListSignatureStatus",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Contract ID",
+            "description": "ID of the contract to list all signatures",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/contracts/{id}/transaction/recipients/{cfc_id}/relaunch": {
+      "get": {
+        "summary": "Resend notification email to a recipient",
+        "description": "Send a notification email to a specific recipient for contract",
+        "operationId": "ResendNotificationRecipient",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Contract ID",
+            "description": "ID of the contract to resend notification",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "cfc_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Recipient for Contract ID",
+            "description": "ID of the recipient for contract object to send a notification",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Specify if the notification was successfuly resend"
```

Comment:
@QwerioTEAM all files should be free of grammatical errors. please change "successfuly" to "successfully"